tests/conftest.py

```diff
@@ -164,38 +166,238 @@ def selenium_chrome_driver(request, live_server):
 
 
 @pytest.fixture(scope="function")
-def mock_get_banner_state(mocker, scope="function"):
+def mock_get_banner_state(request):
     from app.assess.models.banner import Banner
 
+    marker = request.node.get_closest_marker("application_id")
+    application_id = marker.args[0]
+
     mock_banner_info = Banner.from_filtered_dict(
-        {
-            "short_id": "short",
-            "project_name": "name",
-            "funding_amount_requested": 10,
-            "workflow_status": "IN_PROGRESS",
-            "fund_id": "funding-service-design",
+        mock_api_results[
+            "assessment_store/sub_criteria_overview/"
+            f"banner_state/{application_id}"
+        ]
+    )
+
+    with (
+        mock.patch(
+            "app.assess.helpers.get_banner_state",
+            return_value=mock_banner_info,
+        ),
+        mock.patch(
+            "app.assess.routes.get_banner_state",
+            return_value=mock_banner_info,
+        ),
+    ):
+        yield
+
+
+@pytest.fixture(scope="function")
+def mock_get_fund():
+    from app.assess.models.fund import Fund
+
+    mock_fund_info = Fund.from_json(
+        mock_api_results["fund_store/funds/{fund_id}"]
+    )
+
+    with (
+        mock.patch("app.assess.routes.get_fund", return_value=mock_fund_info),
+        mock.patch("app.assess.helpers.get_fund", return_value=mock_fund_info),
+    ):
+        yield
+
+
+@pytest.fixture(scope="function")
+def mock_get_round():
+    from app.assess.models.round import Round
+
+    mock_fund_info = Round.from_dict(
+        mock_api_results["fund_store/funds/{fund_id}/rounds/{round_id}"]
+    )
+
+    with mock.patch(
+        "app.assess.routes.get_round", return_value=mock_fund_info
+    ) as mocked_round:
+        yield mocked_round
+
+    mocked_round.assert_called_once_with(
+        Config.COF_FUND_ID, Config.COF_ROUND2_ID
```

Comment:
yeah thats a good spot, I just kept what we had in the test but when multifund is implemented this fixture will need refactoring. I held off for now because im unsure what that will look like.